### ISSUE TITLE
Getting issue 'Failed to resolve dependencies'

### ISSUE THREAD

**Issue by hemant89** [Shopify/mobile-buy-sdk-ios-sample#5]
Not able to build and run this sample, getting error related to dependencies:

'because mobile-buy-sdk-ios >=5.1.0 contains incompatible tools version and root depends on mobile-buy-sdk-ios 5.1.0..<6.0.0, version solving failed.'

Is there any issue related to iOS Mobile Buy SDK version?

Cheers!

**Comment by Mako96** [comment]
Yeah I am getting the same issue, any luck with fixing it @hemant89 ?

**Comment by SivamAgarwalla** [comment]
Facing the same issue above, when I try building the sample application, the build is failing and I'm getting:

'because mobile-buy-sdk-ios >=5.1.0 contains incompatible tools version and root depends on mobile-buy-sdk-ios 5.1.0..<6.0.0, version solving failed.'

Would we have to manually update the version of the mobile-buy-sdk-ios in the project? If so how would you go about this? 

Any progress or workarounds on this issue? @Mako96 @hemant89
